packages/analysis/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@terra-ui-packages/analysis",
```

Comment:
Opinions on names? This matches the existing "analysis" folder, but could also be "interactive-analysis", etc (because interactive analysis more precisely describes the feature area, not because of the team name).

Reply:
I think `analysis` is fine. It's closest to the name of the tab where all the functionality lives